### PR TITLE
Add support for pre-commit

### PR DESCRIPTION
**Pull request by mondeja**
As was added in `markdownlint-cli`, I've reused the same configuration to allow support for `pre-commit` hook.

### CODE REVIEW

File: .pre-commit-hooks.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+-   id: markdownlint-cli2
+    name: markdownlint-cli2
+    description: "Checks the style of Markdown/Commonmark files."
```
Comment (DavidAnson) [inline .pre-commit-hooks.yaml:3]: "CommonMark"

File: .pre-commit-hooks.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+-   id: markdownlint-cli2
```
Comment (DavidAnson) [inline .pre-commit-hooks.yaml:1]: 2 space indents, please (as with .workflows)

File: .pre-commit-hooks.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+    name: markdownlint-cli2
+    description: "Checks the style of Markdown/Commonmark files."
+    entry: markdownlint-cli2
+    language: node
+    types: [markdown]
+    minimum_pre_commit_version: 0.15.0
```
Comment (DavidAnson) [inline .pre-commit-hooks.yaml:7]: Why is this needed?

**Review by DavidAnson — CHANGES_REQUESTED** [review]
How do we test this? Can it be automated? Is there a GitHub Action for testing?

Thanks for the contribution!

**Comment by DavidAnson** [comment]
I think I know how to test this if there isn't something ready-made.

**Comment by DavidAnson** [comment]
Don't worry about testing this, I have something ready.

File: .pre-commit-hooks.yaml (hunk 3 above)
Comment (mondeja) [reply .pre-commit-hooks.yaml:7]: `types` property [has been added in version `0.15.0`](https://github.com/pre-commit/pre-commit/blob/master/CHANGELOG.md#0150---2017-07-02).

**Review by DavidAnson — APPROVED** [review]
Looks good, thanks! I can't merge from my phone because the base branch is main instead of next. I'll do that from my computer later today.
